### PR TITLE
Updated test script to generate a single binary

### PR DESCRIPTION
**Pull request by thomasfanell**
* The test script now generates a use-file with all specified tests, which rock compiles into a single binary
* Modified Fixture to set a global flag if one or more tests failed
* Tests that are not extending Fixture now prints a TODO about it. (this is not auto-generated)
* Modified .gitignore to ignore the generated use file

There are no changes to the actual usage of ```test.sh``` from a user point of view.

### CODE REVIEW

**Comment by sebastianbaginski** [comment]
Can we still generate one binary for each test case ? I like it for debugging.

**Comment by thomasfanell** [comment]
No, but maybe we could add a script for that? @simonmika @davidhesselbom?

File: test/base/ByteBufferTest.ooc [hunk 1]
```diff
@@ -1,7 +1,3 @@
 use ooc-base
 
-b := ByteBuffer new(4000)
-b referenceCount decrease()
-b = ByteBuffer new(4000)
-b referenceCount decrease()
-ByteBuffer clean()
+"ByteBufferTest [TODO: Not implemented as a fixture!]" printfln()
```
Comment (davidhesselbom) [inline test/base/ByteBufferTest.ooc]: `printfln`?

File: test/base/DebugPrintTest.ooc [hunk 2]
```diff
@@ -24,3 +24,5 @@ version(debugTests) {
 
 	Debug print("TEST2", 3)
 }
+
+"DebugPrintTest [TODO: Not implemented as a fixture!]"
```
Comment (davidhesselbom) [inline test/base/DebugPrintTest.ooc]: Is this supposed to print something?

**Comment by thomasfanell** [comment]
@sebastianbaginski you can still select a single file though, same as before. ```./test.sh io/CsvFileWriter```

File: test/base/TimerTest.ooc [hunk 3]
```diff
@@ -58,3 +58,5 @@ t stop()
 t start()
 for (i in 0 .. 8) { }
 t stop()
+
+"TimerTest [TODO: Not implemented as a fixture!]"
```
Comment (davidhesselbom) [inline test/base/TimerTest.ooc]: And this?

**Comment by sebastianbaginski** [comment]
@thomasfanell That's great, thanks !

File: test/base/DebugPrintTest.ooc (hunk 2 above)
Comment (thomasfanell) [inline test/base/DebugPrintTest.ooc]: Oh shoot, yes. Forgot the printfln(). :(

**Comment by thomasfanell** [comment]
Sorry for the flag toggling. There might be a quirk I have to sort out.

**Pull request merged**